src/System.Private.Uri/tests/FunctionalTests/UriRelativeResolutionTest.cs

```diff
@@ -138,7 +135,6 @@ public void Uri_Relative_BaseVsSimplePartialPathWithScheme_ReturnsPartialPathWit
             Assert.Equal(expectedResult, resolved.ToString());
         }
 
-        [ConditionalFact(nameof(IsNetCoreOrIsNetfx472OrLater))]
```

Comment:
Removing this makes the test not run.  Why are we doing this?

And if we don't want the test to run, shouldn't the rest of the test method be removed as well?